doc/content/introduction/getting-started/debian-curl_tutorial/qmstr.yaml

```diff
@@ -0,0 +1,36 @@
+project:
+  name: "debian cURL packaging"
+  metadata:
+    Vendor: "Endocode"
+    OcFossLiaison: "Mirko Boehm"
+    OcComplianceContact: "foss@endocode.com"
+  server:
+    rpcaddress: ":50051"
+    dbaddress: "localhost:9080"
+    dbworkers: 4
+  analysis:
+    - analyzer: spdx-identifier-analyzer
+      name: "Simple SPDX Analyzer"
+      config:
+        workdir: "/buildroot"
+    - analyzer: scancode-analyzer
+      name: "Scancode Analyzer"
+      config:
+        workdir: "/buildroot"
+        resultfile: "/buildroot/scancode.json"
+        #cached: "true"
+    - analyzer: test-analyzer
+      name: "Simple CI Test Analyzer"
+      config:
+        workdir: "/buildroot"
+  reporting:
+    - reporter: test-reporter
+      name: "Test Reporter"
+      config:
+        siteprovider: "Endocode"
+    - reporter: qmstr-reporter-html
+      name: "HTML Reporter"
+      config:
+        #generatehtml: "no"
```

Comment:
Looks like a placeholder? If it is not required for the use case, please remove it.